source/_docs/guides/lando.md

```diff
@@ -0,0 +1,148 @@
+
+1. Open your terminal and get to the directory where you plan to run your fancy Lando WP site.
+1. While in your terminal simply type `lando init`.
+
+  ![alt text](https://lh4.googleusercontent.com/wSSyn5FZsxUY_03N-d5vszYVQPljZCBafwsk_kjWP4SJ62QzSvB-YaFrHwX9nZvXD3Lab5LHGEDMUt19JqgFOrBY-nU5UulHgcYYRMnJ "Install Lando to your working directory")
+
+
+	You may choose to start from your current directory or you can clone from Pantheon, Github, or another remote repository, or zip file. For this step, we’ll choose Current Working Directory.  
+
+1. Next, you’ll choose WordPress as your starting recipe.
+
+  ![alt text](https://lh3.googleusercontent.com/9oaWMtWpGPftZ15xrjwB9mcPpFwXNgoS5yUeafZI_hoj57dWS2LLKi6Xfc4odz1yDBfLoatRLkcWEWviyvS0saQSEMJ_BffcolQ2kigA "Install Lando to your working directory")
+
+1.  For your webroot, you may press enter and Lando will spin up the site from your current directory.
+
+1. Enter the name of the site. Use something you’ll remember!
+
+Your local site has been created but your still not done! You still need to start the app, download WordPress and then install WordPress. Sure, it’s a few steps but the command line practice is good for bone growth and keyboard-punching-dexterity.
+
+1. In your terminal type `lando start`. This starts your lando app and gives you some basic information like your Appserver URLs to access the site in your local browser of choice.
+
+  ![alt text](https://lh6.googleusercontent.com/MDz4wDlDJZeEmuNwRKR7rZJStKPLvPQqUjn3kfK_oEk5c33UcO7tJxghP41IUyBnsaEudIUk5Zua5qb8QX06X5uzI86zXmJhorOXIMVS "Install Lando to your working directory")
+
+  Then type `lando wp core download`.
+
+<div class="alert alert-info" role="alert" markdown="1">
+#### Note {.info}
+This is where we talk a little bit about WP-CLI, Lando, and You. To use WP-CLI commands in Lando, simply prefix your wp command with lando and you’re good to go, Lando will run your WP-CLI command all nice like. 
+</div>
+
+1. The `lando wp core download` command downloads the current stable release of WordPress and unpacks it in your current working directory.
+
+1. Your WP configuration file still needs to be created. You can actually do this from the command line. However, we’re going to go to do this the ol’ fashioned way through the browser.
+
+1. In your browser, enter in the URL given to you above: _http://yourappname.lndo.site_. You’ll notice WordPress’s _Famous Five Minute install_ screen. 
+
+1. Click next after choosing your preferred language and click Let’s Go on the following screen.
+
+1. In Lando, for a WordPress installation without Pantheon, the Database, Username, and Password are all `WordPress` and Database Host is `database` (the database host is case-sensitive). Enter your credentials and complete the installation process! Congrats! You now have Lando installed with WordPress! 
```

Comment:
This is where my testing failed. I used the values provided but was unable to connect to the database.